service/test/cmd/typecheckrpc.go

```diff
@@ -0,0 +1,201 @@
+package main
```

Comment:
This checks that calls to the API in `service/rpc2/client.go` (using `rpc2.(*Client).call`) match what's expected by `service/rpc2/server.go`, the compiler doesn't because they are RPC calls.